src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -83,7 +83,7 @@ export const Section: React.FC<SectionProps> = ({ venue }) => {
 
   return (
     <div className="Section">
-      <BackButton onClick={backToMain} locationName="overview" />
+      <BackButton onClick={backToMain} />
```

Comment:
Why was `locationName` removed here? We want to keep this on the main platform, it should only be removed on `env/github`.

Reply:
fixed in 088c807569a37b13ce953e79d623431827421b6a